### PR TITLE
update commit checker to tolerate glide updates, clarify bump commit

### PR DESCRIPTION
**Pull request by liggitt**

### CODE REVIEW

**Comment by liggitt** [comment]
travis was failing on `bump (*)` vs `bump(*)` in https://travis-ci.org/openshift/origin/builds/406293134?utm_source=github_status&utm_medium=notification

File: tools/rebasehelpers/util/git.go [hunk 1]
```diff
@@ -118,6 +118,8 @@ type File string
 func (f File) HasVendoredCodeChanges() bool {
 	return strings.HasPrefix(string(f), "Godeps/_workspace") ||
 		strings.HasPrefix(string(f), "vendor") ||
+		strings.HasPrefix(string(f), "glide.lock") ||
+		strings.HasPrefix(string(f), "glide.yaml") ||
```
Comment (deads2k) [inline tools/rebasehelpers/util/git.go]: does this mean that I'd have to put glide.yaml changes into a bump commit?

Comment (liggitt) [reply tools/rebasehelpers/util/git.go]: it didn't have that effect when I tested it, it just tolerated it in a bump commit, but now I'm not sure... I'll drop this... the other bits were more important

**Pull request merged**